src/System.Configuration.ConfigurationManager/ref/System.Configuration.cs

```diff
@@ -1596,6 +1596,16 @@ public partial class ProviderException : System.Exception
         public ProviderException(string message, System.Exception innerException) { }
     }
 }
+namespace System.Drawing.Configuration
+{
+    public sealed partial class SystemDrawingSection : System.Configuration.ConfigurationSection
```

Comment:
@mellinoe is this even interesting to port? It doesn't do anything so I wonder what the value of having it is.